apis/apps/v1alpha1/cluster_types.go

```diff
@@ -296,9 +292,12 @@ type ReplicationMemberStatus struct {
 type ClusterSwitchPolicy struct {
 	// TODO other attribute extensions
 
-	// clusterSwitchPolicy type defined by Provider in ClusterDefinition, refer components[i].replicationSpec.switchPolicies[x].type
+	// clusterSwitchPolicy defines type of the switchPolicy when workloadType is Replication.
+	// MaximumAvailability: [WIP] when the primary is active, do switch if the synchronization delay = 0 in the user-defined lagProbe data delay detection logic, otherwise do not switch. The primary is down, switch immediately. Subsequent versions will soon support.
+	// MaximumDataProtection: [WIP] when the primary is active, do switch if synchronization delay = 0 in the user-defined lagProbe data lag detection logic, otherwise do not switch. If the primary is down, if it can be judged that the primary and secondary data are consistent, then do the switch, otherwise do not switch. Subsequent versions will soon support.
```

Comment:
is `MaximumAvailability` & `MaximumDataProtection` still WIP? remove from enum option until it's ready?

Reply:
> is `MaximumAvailability` & `MaximumDataProtection` still WIP? remove from enum option until it's ready?

enums removes done